backend/kernelCI_app/urls.py

```diff
@@ -5,5 +5,6 @@
 urlpatterns = [
     path('tree/', views.TreeView.as_view(), name='tree'),
     path('tree/<str:commit_hash>', views.TreeDetails.as_view(), name='treeDetails'),
-    path('build/<str:build_id>', views.BuildDetails.as_view(), name='buildDetails')
+    path('build/<str:build_id>', views.BuildDetails.as_view(), name='buildDetails'),
+    path('build/test/<str:build_id>', views.BuildTests.as_view(), name='buildTests'),
```

Comment:
nit: I would put `test` after `build_id` because this is the tests of a specific build_id
I would also pluralize `test` to `tests`